docs/operations_tasks/Ingest-Release-SOP.md

```diff
@@ -8,73 +8,79 @@ parent: Operations tasks
-> Note: You will need to coordinate with other users as it is a shared environment.
-- You may do this for many reasons, including:
-    - Manual testing of microservices interactions
-    - Running automated end-to-end tests
-    - Getting wrangler feedback during development if needed
-3. Create a PR against master
-    - Indicate the ZenHub ticket (If no related Zenhub ticket, make this explicit in the PR)
-    - Ensure Tests Passing
-        - Unit Tests
-        - Per-Component Tests
-        - End-to-End Tests
-    - Requires 1-2 reviewers approval
-    - Code coverage should not go down
-1. Merge to master.
-1. After merge. Tag the merge commit.
-1. Deploy the tagged image to Dev.
-1. Make sure all tests are passing (unit, ingest integration/e2e tests).
-    - Merges that result in failed End-To-End tests should be fixed in `30 minutes`.
-    - Developers should collaborate to fix the issue if possible or make the decision to roll back.
-1. Release to Staging
-
-### Releasing to staging and prod
-1. Coordinate with wranglers who may be using the server
-    - Give a heads up during stand up meeting.
-2. Update changelog.
-    - If we have some extra instructions for deployment, we could note it in the changelog (not really sure where to put this).
-3. Deploy tagged master branch
-4. Check for failed end-to-end tests
-5. Wrangler acceptance testing
-6. For production release, add version tag dYYYY-MM-DD.<release-count> (e.g. d2020-03-08.1).
-    - If there were 2 releases for that day, the second tag will be d2020-03-08.2
-7. Send message to AIT channel #hca when
-    - release is starting
-    - release is completed + release notes
-
-#### Release schedule
-- Staging Release : 10am every tues before sprint demo
-- Prod Release : 10am every tues mid sprint
-
-**Adhoc releases can happen and must be agreed upon by the team.**
-
-### Releasing hotfixes
-1. Inform the team of the hotfix. Team will decide if hotfix is needed and safe.
-2. Branch from the commit hash currently deployed in staging.
-3. Tag the commit. Deploy to staging. All tests must passed (DCP tests in staging, ingest tests, unit tests)
-5. Make a hotfix release notes
-7. Deploy to prod. Make sure all tests passed after hotfix release.
+
+### Pushing changes from dev, through staging, into production
+1. Create feature branch based from `dev`
+2. Commit your changes
+3. Create a pull request (PR) to `dev`
+4. Ensure unit tests and build steps are passing in [GitLab](https://gitlab.ebi.ac.uk/hca) for the project you're working on
+    - These will be reported in the PR UI in Github
+5. Get approvals from two other devs on the PR
+6. Merge PR
+7. Ensure deploy and integration tests in `dev` environment are passed in GitLab for the project
+8. Merge `dev` to `master`
+    - `git checkout dev && git pull && git checkout master && git pull && git merge dev && git push`
+9. Ensure deploy and integration tests in `staging` environment are passed in GitLab for the project
```

Comment:
Can a link be added here? or a notice about the slack channel, and how to join it?